pkg/k8s/osm_injector.go

```diff
@@ -0,0 +1,22 @@
+package k8s
+
+import (
+	"context"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/openservicemesh/osm/pkg/injector"
+)
+
+// GetOSMInjectorPods returns a list of osm-injector pods in the namespace
+func GetOSMInjectorPods(clientSet kubernetes.Interface, ns string) *corev1.PodList {
+	labelSelector := metav1.LabelSelector{MatchLabels: map[string]string{"app": injector.InjectorServiceName}}
```

Comment:
Hi @shashankram - importing `pkg/injector` causes a circular dependency. Would it be better to have the injector constant be in `pkg/constants` for the sake of consistency with the osm-controller constant plus to prevent circular dependencies?

Reply:
Sure that's fine if it is resulting in a circular dependency.